modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-webengine/src/main/resources/skin/resources/script/jquery.toc.js

```diff
@@ -0,0 +1,25 @@
+(function ($) {
+
+  $.fn.toc = function (tocListContainer, tocList) {
+    $(tocList).empty();
+
+    var index = 0;
+    $("h2").each(function () {
```

Comment:
Apparently `jquery#each` callback already contains index and obj ref: https://api.jquery.com/jQuery.each/#jQuery-each-object-callback

Reply:
even if i need the index outside of the iteration?

Reply:
you mean the list size?

Reply:
i mean the number of toc items that were processed

Reply:
i admit i originally copy/pasted the whole code, so i can rename the variable if it makes things clearer

Reply:
Removed the use case after all (always show the TOC even if empty)